src/common/dataHooks/useDictionaryEntry.js

```diff
@@ -14,11 +14,13 @@ import {
   entryForApi,
 } from 'common/dataAdaptors/dictionaryAdaptors'
```

Comment:
I'm fine with these blanket changes in the hooks if @gmcauliffe is, but just as another option...

Another way to handle this situation where there are two similar use cases with different parameters is to pull the common logic into a second hook. So e.g., useDictionaryEntry could get the sitename from useParams and use that sitename to call a new, second hook called something like useDictionaryEntryCrossSite(sitename). 

The benefits are that the consumers of the existing hook don't have to change, and each hook function sticks to the single responsibility principle. It's easier to reason about and test the code if each function can only be called one way and has fewer code paths. E.g., right now if someone meant to specify the sitename parameter but messed up, it could default to a value that might be wrong but wouldn't throw an error. That kind of bug is hard to catch. If the two use cases have to call two different functions, it's easier to spot when you are using the wrong one and you will get an error if you try to use them wrong.

Reply:
I'm inclined to agree with @sahil97's approach and rationale here.

Reply:
Having now looked through the rest of the PR, these modifications to the hooks for the individual entry types (including songs etc) seems like a reasonable solution. I'd be fine for it to stay as is. 👍

Reply:
Another approach could be to always pass the sitename from the parent and never set it inside these hooks. Could offset the "hard to find" bug possibility but would take further code changes in the places that these hooks are being used.